source/modules/max7c0/ublox7rx.h

```diff
@@ -0,0 +1,32 @@
+#ifndef UBLOX7RX
+#define UBLOX7RX 
+
+#include<stdio.h>
+#include<string.h>
```

Comment:
if you don't an include in the header file, it shouldn't be included (same comment for stdio.h)